### ISSUE TITLE
structured bindings squiggle on MinGW/WSL - cannot instantiate std::tuple_element

### ISSUE THREAD

**Issue by bobbrow** [microsoft/vscode-cpptools#2474]
**Type: LanguageService**

**To Reproduce**
Try to use structured bindings in gcc/clang mode on Windows
```
void f()
{
    std::map<std::string, std::string> m;
    for (auto [key, value] : m)
    {
        std::cout << key << value << std::endl;
    }
}
```

![image](https://user-images.githubusercontent.com/12818240/44928556-0b443680-ad0d-11e8-8458-bb5bf02dcfe4.png)

This works fine with gcc/clang mode on Linux so it could just be a config issue.

**Comment by bobbrow** [comment]
Related: #2472

**Comment by sean-mcmanus** [comment]
A user reported this issue at https://github.com/Microsoft/vscode-cpptools/issues/2774 . Our parser has a problem parsing the gcc tuple header. I wasn't able to get an isolated repro.

UPDATE: This was fixed with the [0.21.0-insiders2](https://github.com/Microsoft/vscode-cpptools/releases)
